### PR TITLE
Steve

### PR DESCRIPTION
**Pull request by SomaSapien**
Updated database functionality, structs and variables to make compatible with the current database design.
Archived old / incompatible code to "_old" folder.
Updated main.go accordingly.
Folder structure modified / cleaned slightly.

### CODE REVIEW

**Review by Mainm0e — APPROVED** [review]
Approve

**Comment by maryamkheirkhah** [comment]
good job

File: backend/db/db_structs.go
Comment (maryamkheirkhah) [inline backend/db/db_structs.go:1]: for users we need a private column too, but i can add it later when i merge my branch

**Review by maryamkheirkhah — APPROVED** [review]
good job

**Pull request merged**